clifford/test/test_trig_functions.py

```diff
@@ -0,0 +1,79 @@
+
+from .. import Cl
+import numpy as np
+import pytest
+
+
+class TestScalarProperties:
```

Comment:
Can you test this for complex numbers too using `cmath`?

Reply:
(you can convert from Multivectors in that algebra to complex with something like `mv.value.astype(float).view(complex)[0]`